database/migrations/2020_03_18_173706_lti_connection_content_id.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class LtiConnectionContentId extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('lti_connections', function (Blueprint $table) {
+            $table->string('lti_content_id')->nullable()->after('lti_consumer_key');
```

Comment:
that's not a "lti id for the content" (the lti context_id is), but a content id from our platform to be linked with this LTI connection. So I think the attribute should be "content_id" instead of "lti_content_id"

Reply:
fixed